locales/nl_NL.json

```diff
@@ -1,109 +1,109 @@
 {
   "sandbox-warning": "This is a sandbox environment for testing API integration. Never send real funds here!",
-  "buy": "buy",
-  "sell": "sell",
-  "continue": "Continue",
+  "buy": "koop",
+  "sell": "verkoop",
+  "continue": "verder",
   "ok": "OK",
-  "back": "Back",
+  "back": "Terug",
   "help": "Help",
-  "and": "and",
-  "from": "From",
-  "to": "To",
-  "or": "or",
-  "price": "Price",
-  "fee": "Fee",
-  "try-again": "Try again",
-  "refresh": "Refresh",
-  "finish": "Finish",
-  "price-each": "{{price}} each",
+  "and": "en",
+  "from": "Van",
+  "to": "Naar",
+  "or": "of",
+  "price": "Prijs",
+  "fee": "Servicekost",
```

Comment:
"Servicekosten" of eventueel "commissie".

Reply:
"Servicekosten" gebruiken ze bij Anycoin.eu. maar 'Commissie' lijkt me inderdaad correcter!